Demo/Application/Features/PayPal - Credit/BraintreeDemoPayPalCreditPaymentViewController.m

```diff
@@ -49,12 +49,12 @@ - (void)tappedPayPalOneTimePayment:(UIButton *)sender {
     [sender setEnabled:NO];
 
     BTPayPalDriver *driver = [[BTPayPalDriver alloc] initWithAPIClient:self.apiClient];
-    BTPayPalCheckoutRequest *request = [[BTPayPalCheckoutRequest alloc] initWithAmount:@"4.30"];
-    request.activeWindow = self.view.window;
-
-    request.offerCredit = YES;
 
     if (self.paypalTypeSwitch.selectedSegmentIndex == 0) {
+        BTPayPalCheckoutRequest *request = [[BTPayPalCheckoutRequest alloc] initWithAmount:@"4.30"];
```

Comment:
I wonder if we should rename this to `BRaintreeDemoPayPalPayLaterPaymentViewController`?

Reply:
Maybe let's save that for a later PR.